designer/client/src/http/HttpService.ts

```diff
@@ -341,11 +343,16 @@ class HttpService {
     deploy(
         processName: string,
         comment?: string,
+        nodesDeploymentData?: NodesDeploymentData,
     ): Promise<{
         isSuccess: boolean;
     }> {
+        const runDeploymentRequest = {
```

Comment:
Can't we just simplify it to

const runDeploymentRequest = { nodesDeploymentData, comment };
?

Undefined values will be removed automatically from a JSON.

Alternatively, if we want to do it explicitly, maybe can we use lodash?

const runDeploymentRequest = _.pickBy({comment, nodesDeploymentData});

Reply:
The concept was to "append" property with key "comment" and value of comment when value of comment is defined. 

E.g. to keep clean deploy request, in `{ comment: someValue }` I want to have either `{ comment: "this is text" }` or no propery at all  when undefined `{  }`.

Reply:
yes, I just wanted to point out that we have two IMO more readable approaches to it, explicitly with lodash or implicitly with automatically removing undefined properties by JSON

Reply:
Fixed (the first option without lodash) and tested, works perfectly, thanks for the hint :)